test/fixtures/api/prod.json

```diff
@@ -0,0 +1 @@
e no locations with available appointments within 50 miles of Leominster, MA 01453  .\",\"signUpLink\":\"https://hannafordsched.rxtouch.com/rbssched/program/covid19/Patient/Advisory\",\"street\":\"118 Lancaster St.\",\"city\":\"Leominster\",\"zip\":\"01453\"},{\"name\":\"Southbridge Community Center\",\"street\":\"153 Chestnut Street\",\"city\":\"Southbridge\",\"zip\":\"01550\",\"website\":\"https://signup.com/api/events?accesskey=6775657374&activity_id=3500438&enddate=2021%2F12%2F31&include_comments=false&include_jobassignments=true&include_jobs=true&my_jobs=false&selected_activity=3500438&startdate=2020%2F12%2F16\",\"signUpLink\":\"https://forms.gle/tMDnRLcXzZnPUxqP9\",\"availability\":{\"2/15/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/16/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/17/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/18/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/19/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/22/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/23/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/24/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/25/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"2/26/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/1/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/2/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/3/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/4/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/5/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/8/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/9/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/10/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/11/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/12/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/15/2021\":{\"numberAvailableAppointments\":1,\"hasAvailability\":true},\"3/16/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/17/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/18/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/19/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/22/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/23/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/24/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/25/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false},\"3/26/2021\":{\"numberAvailableAppointments\":0,\"hasAvailability\":false}},\"hasAvailability\":true}]}"}
```

Comment:
I think it would be more useful for testing purposes (and readability) if we formatted this like a regular JSON file and cut the results down to fewer locations. Maybe one with no availability, one that has multiple sign-up links with a drop-down (you can find an example of this when you see a signUpLink for each date of availability), one with multiple sign-up links but only one date available, and one with a single signUpLink.

Reply:
I can remove some of the appointment entries from the results array, no problem.

but this file is already formatted as a regular JSON file; its contents are exactly what the browser receives when a request to the `/prod` API endpoint is made.  if you're specifically referring to the string of JSON stored in the `body` field, then I think there's value in keeping mock data identical to what's returned by a real API call but if you want I can store a decoded version of that string here in this file and re-encode it inside the unit tests (since the code is expecting a string, not an object).

Reply:
That would be awesome if you could re-encode it like you suggested! That way we can add some indentation and new lines for readability which would be A+